### PR TITLE
Review ZeroHero Tutorial

### PR DESCRIPTION
**Pull request by fiexer**
Made sure all tutorial match.

### CODE REVIEW

File: docs/builder-guides/from-zero-to-ink-hero/nft/Events/events.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-## Minting event
+## Minting Event
 In our contract there is one occasion where an event should be emitted and that is when token is minted.
-One could expect that by calling Openbrush `_mint_to()`, and event will be emitted but upon closer examination we can see that `_emit_transfer_event()` has empty default [implementation](https://github1s.com/Supercolony-net/openbrush-contracts/blob/main/contracts/src/token/psp34/psp34.rs#L151-L152). This gives flexibility to create events for our needs.
+One could expect that by calling Openbrush `mint_to()`, and event will be emitted but upon closer examination we can see that `emit_transfer_event()` has empty default [implementation](https://github1s.com/Supercolony-net/openbrush-contracts/blob/main/contracts/src/token/psp34/psp34.rs#L151-L152). This gives flexibility to create events for our needs.
```
Comment (Maar-io) [inline docs/builder-guides/from-zero-to-ink-hero/nft/Events/events.md:9]: actually both functions do start with underscore _

File: docs/builder-guides/from-zero-to-ink-hero/nft/Events/events.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 ```rust
 default fn _emit_transfer_event(&self, _from: Option<AccountId>, _to: Option<AccountId>, _id: Id) {}
 ```
-Let's define two events that are needed for token handling, *Transfer* and *Approve*. This needs to be done in the contracts's `lib.rs`. Please note that there is no Mint event. Mint is covered by *Transfer* event where `from` will be the contract address.
+Let's define two events that are needed for token handling, *Transfer* and *Approve*. This needs to be done in the contracts's `lib.rs`. Please note that there is no mint event. Mint is covered by *Transfer* event where `from` will be the contract address.
```
Comment (Maar-io) [inline docs/builder-guides/from-zero-to-ink-hero/nft/Events/events.md:14]: events are usually with capital letters. maybe use back tick

File: docs/builder-guides/from-zero-to-ink-hero/nft/PayableMintTrait/payableminttrait.md [hunk 3]
```diff
@@ -3,30 +3,30 @@ So far our `mint()` method was very generic giving a freedom to a caller to mint
...
-### mint(to: AccountId, mint_amount: u64)
-The mint() method now takes receiver nft account and amount of tokens to be minted.
+### `mint(to: AccountId, mint_amount: u64)`
+The `mint()` method now takes receiver NFT account and amount of tokens to be minted.
 This will allow contract to be in charge which token will be minted next and allows minting more than one token at the time.
 
-### set_base_uri(uri: PreludeString)
-First we need to import String from `ink_prelude` and rename it not to be mixed with Openbrush String implementation. The difference is that Openbrush String is in fact a vector of u8 elements. Since we expect users to use utf-8 string we use String from prelude.
+### `set_base_uri(uri: PreludeString)`
+First we need to import string from `ink_prelude` and rename it not to be mixed with Openbrush String Implementation. The difference is that Openbrush String is in fact a vector of u8 elements. Since we expect users to use `utf-8` string we use String from prelude.
```
Comment (Maar-io) [inline docs/builder-guides/from-zero-to-ink-hero/nft/PayableMintTrait/payableminttrait.md:11]: string is type. Should be capital. maybe with back ticks

File: docs/builder-guides/from-zero-to-ink-hero/nft/Wizard/wizard.md [hunk 4]
```diff
@@ -150,19 +150,20 @@ And your folder structure will look like:
...
 cargo check
 ```
 
-## Examine Openbrush traits 
-Let's examine what we have inside module shiden34 (lib.rs) so far:
-* Defined structure `Contract` for contract storage
-* Implemented constructor `new()` for `Contract` structure
-* Implemented Openbrush traits *PSP34, Metadata, Mintable, Enumberable, Ownable* for structure `Contract`
+## Examine Openbrush Traits 
+Let's examine what we have inside module `Shiden34` (lib.rs) so far:
```
Comment (Maar-io) [inline docs/builder-guides/from-zero-to-ink-hero/nft/Wizard/wizard.md:160]: shiden34 is module name. no caps

**Review by PierreOssun — APPROVED** [review]
LGTM

**Pull request merged**